index.md

```diff
@@ -0,0 +1,8 @@
+--- copyright: years: 2021 - 2023 lastupdated: "2023-04-18" layout: page title: "Home" description: "Welcome to the Open Horizon Documentation web site" nav\_order: 1 ---   
```

Comment:
For some strange reason, the front matter is appearing all on one line.  Can you please edit to separate and also remove the escape backslash before the underscore in `nav_order`?